contract-addresses.json

```diff
@@ -11,5 +11,9 @@
   "ProtocolDataProvider" : "0x057835Ad21a177dbdd3090bB1CAE03EaCF78Fc6d",
   "WETHGateway" : "0xcc9a0B7c43DC2a5F023Bb9b738E45B0Ef6B06E04",
   "AaveCollector" : "0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c",
-  "IncentivesController" : "0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5"
+  "IncentivesController" : "0xd784927Ff2f95ba542BfC824c8a8a98F3495f6b5",
+  "AaveGovernanceV2" : "0xEC568fffba86c094cf06b22134B23074DFE2252c",
+  "ExecutorShort" : "0xee56e2b3d491590b5b31738cc34d5232f378a8d5#readContract",
+  "ExecutorLong" : "0x61910ecd7e8e942136ce7fe7943f956cea1cc2f7",
+  "EcosystemReserve" : "0x25f2226b597e8f9514b3f68f00f494cf4f286491"
```

Comment:
these addresses aren't checksummed. Also you copied the #readcontract from the URL in etherscan. might be best to press the copy button to get the checksummed addresses

Reply:
Fixed